src/UI/Component/MessageBox/Factory.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/* Copyright (c) 2018 Thomas Famula <famula@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\MessageBox;
+
+/**
+ * This is how a factory for Message Boxes looks like.
+ */
+interface Factory {
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     The system failed to complete some actions and displays information about the failure.
+	 *   composition: >
+	 *     The div uses the alert-danger Bootstrap style.
```

Comment:
Very technical, consider refrasing "the alert-warning color is used as background of the message."